composer-env.nix

```diff
@@ -0,0 +1,244 @@
+# This file originates from composer2nix
```

Comment:
See also this - you don't have to worry about this file. `composer2nix` will write out the "correct" version.